app.py

```diff
@@ -0,0 +1,9 @@
+from flask import Flask, request, jsonify
```

Comment:
leave a line to separate the imports from the rest of the code so that it looks clean